rlay-client-lib/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We have `.eslintrc` and `.eslintrc.json` here. I think this usually leads to only one of them being used? Do we need the `.eslintrc.json` which seems very verbose?

Reply:
So, `.eslintrc.json` is the actual config file which the linter needs: https://eslint.org/docs/user-guide/configuring

The problem is which `.eslintrc`, which shouldn't be there and comes from `oclif`. In the coming commit, I will merge the content of `.eslintrc` into the `.eslintrc.json` and then remove the `.eslintrc`.